esmvaltool/preprocessor/__init__.py

```diff
@@ -14,16 +14,16 @@
     cmor_check_metadata
 from ._regrid import vinterp as extract_levels
 from ._regrid import regrid
-from ._time_area import volume_average as average_volume
-from ._time_area import area_average as average_region
-from ._time_area import time_average as time_average
-from ._time_area import area_slice as extract_region
-from ._time_area import time_slice as extract_time
-from ._time_area import volume_slice as extract_volume
-from ._time_area import extract_trajectory as extract_trajectory
-from ._time_area import extract_slice as extract_slice
-from ._time_area import depth_integration as depth_integration
-from ._time_area import seasonal_mean
+from ._time_pp import time_average as time_average
```

Comment:
There's no need for the `as time_average`, as you are using the same name, so this can be removed.